external_search.py

```diff
@@ -641,20 +648,31 @@ def map_fields_by_type(cls, fields_by_type, mapping=None):
             to field names.
         """
         for type, fields in fields_by_type.items():
+            description={
+                "type": type,
+                # TODO: In some cases we can get away with setting
+                # index: False here, which would presumably lead
+                # to a smaller index and faster updates. However,
+                # it might hurt performance of searches. When this
+                # code is more mature we can do a side-by-side
+                # comparison.
+                "index": True,
+                "store": False,
+            }
+            if type == 'icu_collation_keyword':
```

Comment:
For the most part, this lets us pretend that `icu_collation_keyword` works when we're defining fields.